rest/routes/recruitment.py

```diff
@@ -58,3 +61,41 @@ def get_all_recruitment_data():
     data = dict()
     data['data'] = Recruitment.to_json()
     return create_response(data, 200, '*')
+
+
+@recruitment_url.route('/point-limit', methods=['POST', 'OPTIONS'])
+def calculate_point_limit():
+    """
+    Calculates point limit for every recruitment that has ended
+    :return: flask Response with operation result
+    """
+    role, response = handle_request_token(request)
+
+    if role is None:
+        return response
+
+    try:
+        recruitments = Recruitment.query.filter_by(point_limit=None)\
+                                        .filter(Recruitment.end_date <= date.today())
+        for recruitment in recruitments:
+            candidates = CandidateRecruitment.query.filter_by(recruitment_id=recruitment.id)\
+                                                    .order_by(CandidateRecruitment.points.desc())
+            places_left = recruitment.slot_limit
+            point_limit = 0
+            for candidate in candidates:
+                if places_left > 0 and candidate.is_paid:
+                    candidate.status = RecruitmentStatus.QUALIFIED
+                    places_left -= 1
+                    if places_left == 0:
+                        point_limit = candidate.points
+                else:
+                    candidate.status = RecruitmentStatus.NOT_QUALIFIED
+            recruitment.point_limit = point_limit
+
+        db.session.commit()
+    # pylint: disable=broad-except
+    except Exception as exception:
+        print(exception, file=sys.stderr)
+        return create_response({"error": "Nie udało się obliczyć progów."}, 400, "*")
```

Comment:
this type of "try except" is quite risky, add:
```python
except AttributeError as ae:
     print(ae, file=sys.stderr)
```
as exception for bad references

```python
except SQLAlchemyError as e:
  print(e, file=sys.stderr)
```
as exception for SQLAlchemy